server/sonar-server/src/test/java/org/sonar/server/permission/GroupPermissionChangerTest.java

```diff
@@ -53,15 +52,14 @@
 
   private UserSessionRule userSession = UserSessionRule.standalone();
   private GroupPermissionChanger underTest = new GroupPermissionChanger(db.getDbClient(), userSession);
-  private UserDbTester userTester = new UserDbTester(db);
```

Comment:
same comment as above and probably as the rest of the commit : why introduce the duplication of `db.users()` ?